apps/wso2-digital-wallet/src/pages/Profile/Profile.css

```
@@ -7,76 +7,342 @@
+.profile-wallet-info {
   display: flex;
   flex-direction: column;
+  gap: 4px;
+  min-width: 0;
+  flex: 1;
 }
-.wallet-list-item:hover {
-  box-shadow: 0 2px 8px rgba(0,0,0,0.08);
+
+.profile-wallet-addr {
+  font-family: 'JetBrains Mono', 'Monaco', 'Menlo', 'Ubuntu Mono', monospace;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Monaco" (font-family-name-quotes)

(font-family-name-quotes)